docs/sources/reference/commandline/cli.md

```diff
@@ -1674,6 +1682,19 @@ An example of a file passed with `--env-file`
 This will create and run a new container with the container name being
 `console`.
 
+    $ sudo docker run --syslog --syslog-tag MONGODB mongo:latest
+
+The `--syslog` flag will make Docker forward logs to the host system's log server.
```

Comment:
can we just change this to `The `--syslog` flag will make Docker forward logs to the system's log server in addition to sending it to Docker's internal log store.`

please?